addons/web/static/src/views/kanban/kanban_controller.scss

```diff
@@ -106,6 +106,7 @@
 
         .o_kanban_record_top {
             align-items: flex-start;
+            width: 95%;
```

Comment:
You can revert that changes since the following point will be skipped. :) 

```txt
Would it be possible to have a line break a bit more to the left so that the text doesn't overlap with the burger menu?
```